Source/WarpX.cpp

```diff
@@ -1176,52 +1178,57 @@ WarpX::ReadParameters ()
         // Check whether the default comoving velocity should be used
         bool use_default_v_comoving = false;
         pp_psatd.query("use_default_v_comoving", use_default_v_comoving);
+
+        WARPX_ALWAYS_ASSERT_WITH_MESSAGE(
+            !use_default_v_comoving || !boosted_frame,
```

Comment:
Shouldn't this read
```suggestion
            use_default_v_comoving && !boosted_frame,
```
?

Reply:
yes, sorry! You are right!!